CONTRIBUTING.md

```diff
@@ -1,17 +1,70 @@
 # Contributing to RudderStack #
 
-:heart: Thanks for taking the time and for your help improving this project!
+Thanks for taking the time and for your help improving this project!
 
 ## Getting Help ##
 
 If you have a question about rudder or have encountered problems using it,
-start by asking a question on [discord][discord]
+start by asking a question on [discord][discord].
 
 ## Rudder Labs Contributor Agreement ##
 
 To contribute to this project, we need you to sign to [Contributor License Agreement (“CLA”)][CLA] for the first commit you make. By agreeing to the [CLA][CLA]
 we can add you to list of approved contributors and review the changes proposed by you.
 
+## Installing and Setting Up RudderStack
+
+To contribute to this project, you need to install RudderStack on your machine. By following a few simple instructions, you can get your machine up and running to use RudderStack in no time.
+
+
+1. Download and install [Golang 1.13] (https://golang.org/dl/) or above.
+
+2. Download and install [Node.js 10.6] (https://nodej.org/en/download/) or above.
+
+3. Download and install [PostgreSQL 10](https://www.postgresql.org/download/) or above, and set up the database using the following commands:
+
+ ```
+ createdb jobsdb
+ createuser --superuser rudder
+ psql "jobsdb" -c "alter user rudder with encrypted password 'rudder'";
+ psql "jobsdb" -c "grant all privileges on database jobsdb to rudder";
+
+ ```
+
+4. Go to the [RudderStack dashboard] (https://app.rudderlabs.com/signup) and set up your account. Copy your workspace token from the top of the home page.
```

Comment:
Let's change the link to https://app.rudderstack.com/signup ?

Reply:
Agreed. Will incorporate this change.